go/mock_server.go

```diff
@@ -256,8 +258,14 @@ func main() {
 		panic(err)
 	}
 	log.Println("Server started, listening to port", port)
-	// Change this to whatever database you want to use.
-	db, err = gorm.Open("postgres", "host=34.213.216.228 port=5432 user=modelzoo")
+	log.Println("Running in", os.Args[1], "mode.")
+	if os.Args[1] == "public" {
```

Comment:
This is find for now, but in future we should try to use some golang arg parsing libraries.